Mathlib/FieldTheory/IsAlgClosed/Basic.lean

```diff
@@ -74,6 +74,10 @@ namespace IsAlgClosed
 
 variable {k}
 
+/--
+If `k` is algebraically closed, then every nonconstant polynomial has a root.
+-/
+@[stacks 09GR "(4) ⟹ (3)"]
```

Comment:
I was confused at first. Maybe tag `IsAlgClosed` with `09GR` saying that we are using (4) as the official def instead?

Reply:
Also modified the tags before `IntermediateField.eq_bot_of_isAlgClosed_of_isAlgebraic` to @[stacks 09GQ "The result is the definition of algebraically closedness in Stacks Project. \
This statement is 09GR (4) ⟹ (1)."]